lib/Clients/c/.openapi-generator/VERSION

```diff
@@ -0,0 +1 @@
+3.3.4
```

Comment:
Duplicate code . C client library should be under `lib/curl` folder .

Reply:
C client moved to `lib/curl`

Reply:
Really C client was regenerated by openapi-generator with current swagger status in `Skycoin Api`